### ISSUE TITLE
Workflow

### ISSUE THREAD

**Issue by DesislavaMetodieva** [heirfanahmed/mood-movies#15]
User Story:
As a developer working on a project, I want to understand the workflow for development within the project so that I can effectively contribute to the codebase and meet project goals.

Acceptance Criteria:

A clear description of the development process should be outlined, including the steps involved and tools used.
The development environment should be set up and described, including required software and dependencies.

1. Create issue in Repo
2. Add user story and acceptance criteria for each task you're working on, even if it's a research task.
3. Assign it to the assignee
4. Work on the task
5. Test the code to see if all works as expected
6. If all good, push the code to feature branch
7. Ask team mates to review the code before pushing to main branch
8. If all good, push code to main branch
9. Before starting new issue/task, pull the latest file from main
10. Create a local branch following naming convention issue_branch
11. Repeat all steps from 3 - 9

How to assign and create tasks:
Initial tasks are assigned based on the HTML elements
Secondary tasks will be assigned based on the bugs we find in our existing code
Another wave of tasks are to be assigned to do with the CSS for the relevant html elements

The version control system used within the project should be identified, such as Git.
The branching and merging strategy should be outlined, including when and how to create branches and when to merge them into the main branch.
The code review process should be described, including who is responsible for reviewing code and the criteria used for approval.
The testing process should be outlined, including what types of tests are required and how they are run.
The deployment process should be described, including how code changes are moved from development to production.
The documentation process should be outlined, including how to document code and changes to the project.
The project timeline and milestones should be outlined, including deadlines for completing tasks and delivering results.

**Comment by DesislavaMetodieva** [comment]
Updated workflow:

Create issue in Repo
Add user story and acceptance criteria for each task you're working on, even if it's a research task.
Assign it to the assignee

$ git checkout -b HTML-br-Dessy
$ cd mood-movies
$ git checkout -b html-dessy
$ git pull origin main
$ git push origin html-dessy
Work on the task
Test the code to see if all works as expected
If all good, push the code to feature branch
Ask team mates to review the code before pushing to main branch
If all good, push code to main branch
Before starting new issue/task, pull the latest file from main
Create a local branch following naming convention issue_branch
Repeat all steps from 3 - 9
How to assign and create tasks:
Initial tasks are assigned based on the HTML elements
Secondary tasks will be assigned based on the bugs we find in our existing code
Another wave of tasks are to be assigned to do with the CSS for the relevant html elements

Feel free to update as needed :)
